### PR TITLE
✨ Remixd AMP ad-tag implementation

### PR DESCRIPTION
**Pull request by glendza**
The implementation of the ad-tag for the Remixd player.

### CODE REVIEW

File: ads/remixd.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (honeybadgerdontcare) [inline ads/remixd.js:2]: please update year to be 2020.

File: ads/remixd.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (honeybadgerdontcare) [inline ads/remixd.md:2]: please update year to be 2020.

File: ads/_config.js [hunk 3]
```diff
@@ -938,6 +938,10 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'remixd': {
+    preconnect: 'https://tags.remixd.com',
```
Comment (calebcordry) [inline ads/_config.js:942]: Did you have a chance to look at implementing [`renderStart`](https://github.com/ampproject/amphtml/blob/73f4d0eaa7a217ecac0dd4c6181dc4f9297766ef/ads/README.md#available-apis)?  It should give you some performance gains, but ultimately it is your decision?

**Comment by calebcordry** [comment]
Thanks for contributing! Looks like you have a couple of lint errors. You can expose those using `gulp lint`, and try to fix using `gulp lint --fix` or check out more details [here](https://github.com/ampproject/amphtml/blob/df16572d8957be9da58004321e7cbd0d543ae584/contributing/getting-started-e2e.md#code-quality-and-style)

Also I think you need to generate a new validator `.out` file [docs](https://github.com/ampproject/amphtml/blob/532dbbbc15c879a32e26e1e890b3a6b5b71cb123/contributing/component-validator-rules.md#test-output-files) for your new tag. 

Let me know if you need help with the CLA, or feel free to ping me when you get it sorted.

**Comment by glendza** [comment]
@googlebot I signed it!

File: ads/_config.js (hunk 3 above)
Comment (twifkak) [reply ads/_config.js:942]: @glendza Looks like this is an open question for you.

**Review by twifkak — COMMENTED** [review]
@calebcordry Any other comments or approval blockers?

(Hi everybody. Just interjecting because we saw this during a bug scrub.)

File: ads/remixd.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function remixd(global, data) {
+  window._rmxd = {};
```
Comment (calebcordry) [inline ads/remixd.js:22]: please replace all usage of `window` with global

File: ads/remixd.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  window._rmxd.url = data.url || window.context.sourceUrl;
+  const sriptVersion = data.version || '5';
+  const tagUrl = 'https://tags.remixd.com/player/v'+ sriptVersion +'/index.js?cb=' + Math.random();
+
+  document.write(
```
Comment (calebcordry) [inline ads/remixd.js:27]: Please use `loadScript` or `writeScript` from `3p.js` here

**Comment by calebcordry** [comment]
Looks like we still need to sort out the CLA. @glendza let us know if you need assistance. Thanks!

File: ads/remixd.js (hunk 5 above)
Comment (glendza) [reply ads/remixd.js:27]: The reason we are not using writeScript is that the script we need to write needs to have an id set. I saw that writeScript doesn't have an option to set the element id, do you have any other suggestion or can we keep the code like this?

File: ads/remixd.js (hunk 4 above)
Comment (glendza) [reply ads/remixd.js]: Done

**Comment by glendza** [comment]
The CLA was already signed by our CEO, under Remixd Media, Inc. Is there anything else we need to do?

File: ads/_config.js (hunk 3 above)
Comment (glendza) [reply ads/_config.js:942]: Done

File: ads/remixd.md (hunk 2 above)
Comment (glendza) [reply ads/remixd.md]: Done

File: ads/remixd.js (hunk 1 above)
Comment (glendza) [reply ads/remixd.js]: Done

File: ads/remixd.js (hunk 5 above)
Comment (calebcordry) [reply ads/remixd.js:28]: Yeah I guess not, so this is ok. Can you change it to use the passed in `global.document` though? Accessing document directly may do weird things based on different serving contexts.

**Comment by calebcordry** [comment]
re: CLA it looks like your commits are fine, but there are a couple commits in this pr from an author with a `nativo` domain and a `localhost` domain that are not covered under the CLA

You also still have a couple lint errors, and will need to generate the new validator.out files to pass the CI tests.

**Pull request closed without merging**